### PR TITLE
rsync-time-backup 1.1.5 (new formula)

### PR DESCRIPTION
**Pull request by laurent22**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

### CODE REVIEW

**Comment by BenjaminHCCarr** [comment]
I am on El Capitan still and use `hln` for Hard Links as 10.11 didn't ship with `ln` for hard linls only `ln -s` support. This may explain some inconsistencies noted in laurent22/rsync-time-backup#46

What OS version are you testing on? 10.13 with APFS may support Hardlinks out of the box.

You may need to add a minimum OS version of hardlinks were added in Sierra|HighSierra, as HFS+ on 10.11.x doesn't support them natively.

```
benc$ brew info hardlink
hardlink: stable 0.3.0 (bottled)
Replace file copies using hardlinks
https://jak-linux.org/projects/hardlink/
Not installed
From: https://github.com/Homebrew/homebrew-core/blob/master/Formula/hardlink.rb
==> Dependencies
Build: pkg-config ✔
Required: gnu-getopt ✘, pcre ✔
```

The hardlink fomula: (https://github.com/Homebrew/homebrew-core/blob/master/Formula/hardlink.rb)

**Comment by BenjaminHCCarr** [comment]
Okay, now I am more confused:
The code calls for symlinks:
https://github.com/laurent22/rsync-time-backup/blob/master/rsync_tmbackup.sh#L139-L141

But the Readme clearly says you are using Hardlinks:
https://github.com/laurent22/rsync-time-backup/blob/master/README.md#L32

but then references symlinks:
https://github.com/laurent22/rsync-time-backup/blob/master/README.md#L42

**Comment by ilovezfs** [comment]
No idea what you mean @BenjaminHCCarr. HFS+ has support for directory hard links going all the way back to 10.5. Support for directory hard links does not exist in APFS. `ln` is unrelated.

**Comment by laurent22** [comment]
@BenjaminHCCarr, the script uses hardlinks via the rsync flag `--hard-links`. There are places in the script where soft links are used too, but not for the backup itself;

File: Formula/rsync-time-backup.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
+class RsyncTimeBackup < Formula
+  desc "Time Machine-style backup for the terminal, using rsync"
+  homepage "https://github.com/laurent22/rsync-time-backup"
+  url "https://github.com/laurent22/rsync-time-backup/archive/v1.1.5.tar.gz"
+  sha256 "567f42ddf2c365273252f15580bb64aa3b3a8abb4a375269aea9cf0278510657"
+
```
Comment (fxcoudert) [inline Formula/rsync-time-backup.rb]: We need `bottle :unneeded`

**Review by fxcoudert — CHANGES_REQUESTED** [review]

**Comment by laurent22** [comment]
@fxcoudert, I've added `bottle: uneeded`

**Review by fxcoudert — APPROVED** [review]

**Comment by ilovezfs** [comment]
```
rsync-time-backup:
  * C: 6: col 1: Trailing whitespace detected.
```

**Comment by laurent22** [comment]
Sorry about that, I've removed the trailing space

**Review by ilovezfs — APPROVED** [review]

**Comment by ilovezfs** [comment]
Thanks again @laurent22!

**Pull request merged**